full-service/src/lib.rs

```diff
@@ -4,13 +4,14 @@
 // Required because hashbrown is at 0.9.1 and has build issues otherwise
 #![feature(ptr_offset_from)]
 
+mod api;
```

Comment:
This module is to help organize the "API layer" of the application. I think this will be helpful if/when we want to support other API modes (e.g. gRPC), and should help encapsulate where JSON and String-y data are used. I also just like the familiar feel of web applications that are organized around API / Application Logic (Services) / Database (Persistence).